client/store/users/AuthContext.jsx

```diff
@@ -17,21 +17,26 @@ export function AuthProvider({ children }) {
   // add the user to firestore. The postURL function acts as a form of frontend middleware (maybe?) between the
   // frontend and backend. https://www.youtube.com/watch?v=qWy9ylc3f9U
   function signup(uname, uemail, password) {
-    auth.createUserWithEmailAndPassword(uemail, password).then((cred) => {
-      return postURL('/api/users/add', {
-        uid: cred.user.uid,
-        username: uname,
-        email: uemail,
-      });
-    });
+    auth.createUserWithEmailAndPassword(uemail, password).then((cred) => postURL('/api/users', {
```

Comment:
here we don't have a a return before the auth.... but in `loginWithGoogle` we do.  Maybe we should do the same for both?

Reply:
+1 for `return` in case we want to do anything with the `Promise`